jenkins/github_helpers.py

```diff
@@ -59,17 +59,15 @@ def connect_to_repo(github_instance, repo_name):
     """
     Get the repository object of the desired repo.
     """
-    repos_list = github_instance.get_user().get_repos()
-    repository = None
-    for repo in repos_list:
-        if repo.name == repo_name:
-            return repo
-
-    raise Exception(
-        "Could not connect to the repository: {}. "
-        "Please make sure you are using the correct "
-        "credentials and try again.".format(repo_name)
-    )
+    try:
+        repo = github_instance.get_user().get_repo(repo_name)
+    except:
+        raise Exception(
```

Comment:
Maybe log the original exception here so we have access to the original error message and stack trace for debugging?